users/urls.py

```diff
@@ -19,5 +20,10 @@
         CurrentUserRetrieveViewSet.as_view({"get": "retrieve"}),
         name="users_api-me",
     ),
+    path(
```

Comment:
I think this would be better as an extension of the `api/users/me` endpoint by adding the `patch` operation, since the `get` is basically being duplicated here and that goes against typical REST conventions